src/Poseidon/Package.hs

```diff
@@ -75,10 +84,111 @@ data GenotypeDataSpec = GenotypeDataSpec
     deriving (Show, Eq)
 
 -- | A data type representing the options fo the genotype format
-data GenotypeFormatSpec = GenotypeFormatEigenstrat -- ^ the Eigenstrat format
+data GenotypeFormatSpec = 
+      GenotypeFormatEigenstrat -- ^ the Eigenstrat format
     | GenotypeFormatPlink -- ^ the Plink format
     deriving (Show, Eq)
 
+-- |A datatype to represent Genetic_Sex in a janno file
+data Sex = 
+      Male
+    | Female
+    | Unknown
+    deriving (Eq, Show, Ord)
+
+-- |A datatype to represent Date_Type in a janno file
+data JannoDateType = 
+      C14
+    | Contextual
+    | Modern
+    deriving (Eq, Show, Ord)
+
+-- |A datatype to represent Data_Type in a janno file
+data JannoDataType = 
+      Shotgun
+    | A1240K
+    | OtherCapture
+    | ReferenceGenome
+    deriving (Eq, Show, Ord)
+
+-- |A datatype to represent Genotype_Ploidy in a janno file
+data JannoGenotypePloidy = 
+      Diploid
+    | Haploid
+    deriving (Eq, Show, Ord)
+
+-- |A datatype to represent UDG in a janno file
+data JannoUDG = 
+      Minus
+    | Half
+    | Plus
+    | Mixed
+    deriving (Eq, Show, Ord)
+
+-- |A datatype to represent Library_Built in a janno file
+data JannoLibraryBuilt = 
+      DS
+    | SS
+    | Other
+    deriving (Eq, Show, Ord)
+
+-- | A datatype for Latitudes
+data Latitude = 
```

Comment:
In so einem Fall lohnt sich übrigens `newtype` statt `data`. Das sorgt dafür dass der Compiler das intern trotzdem als Double darstellt, nur eben type safe. Ist also performanter. `newtype` lohnt sich immer dann wenn man einen abstrakten Datentypen über einem built-in Datentypen baut, mit einem einfachen Constructor. Nur so als Tipp...